payloads/v8/user.ts

```diff
@@ -85,20 +85,65 @@ export interface APIUser {
  * https://discord.com/developers/docs/resources/user#user-object-user-flags
  */
 export const enum UserFlags {
+	/**
+	 * None
+	 */
 	None = 0,
-	DiscordEmployee = 1 << 0,
-	PartneredServerOwner = 1 << 1,
-	DiscordHypeSquadEvents = 1 << 2,
+	/**
+	 * Discord Employee
+	 */
+	Staff = 1 << 0,
+	/**
+	 * Partnered Server Owner
+	 */
+	Partner = 1 << 1,
+	/**
+	 * HypeSquad Events Coordinator
+	 */
+	Hypesquad = 1 << 2,
+	/**
+	 * Bug Hunter Level 1
+	 */
 	BugHunterLevel1 = 1 << 3,
-	HypeSquadHouseBravery = 1 << 6,
-	HypeSquadHouseBrilliance = 1 << 7,
-	HypeSquadHouseBalance = 1 << 8,
-	EarlySupporter = 1 << 9,
-	TeamUser = 1 << 10,
+	/**
+	 * House Bravery Member
+	 */
+	HypesquadOnlineHouse1 = 1 << 6,
+	/**
+	 * House Brilliance Member
+	 */
+	HypesquadOnlineHouse2 = 1 << 7,
+	/**
+	 * House Balance Member
+	 */
+	HypesquadOnlineHouse3 = 1 << 8,
```

Comment:
Regardless of how the docs name those flags, the correct name is `HypeSquad`, as shown in https://discord.com/hypesquad
```suggestion
	/**
	 * House Bravery Member
	 */
	HypeSquadOnlineHouse1 = 1 << 6,
	/**
	 * House Brilliance Member
	 */
	HypeSquadOnlineHouse2 = 1 << 7,
	/**
	 * House Balance Member
	 */
	HypeSquadOnlineHouse3 = 1 << 8,
```

Reply:
Also apply those changes to v9 and generate Deno's types to match.